content/docs/command-reference/plots/index.md

```diff
@@ -260,3 +277,48 @@ file:///Users/usr/src/plots/classes.csv.html
 
 > A confusion matrix [template](/doc/command-reference/plots#plot-templates) is
 > predefined in DVC (found in `.dvc/plots/confusion.json`).
+
+### Example for the HTML template
+
+Download the Vega libraries into the directory where you'll produce the
+`dvc plots`.
+
+```dvc
+$ wget https://cdn.jsdelivr.net/npm/vega@5.10.0 -O my_vega.js
+$ wget https://cdn.jsdelivr.net/npm/vega-lite@4.8.1 -O my_vega_lite.js
+$ wget https://cdn.jsdelivr.net/npm/vega-embed@6.5.1 -O my_vega_embed.js
+```
+
+Create the following HTML file with your text editor:
+
+```html
+<html>
+  <head>
+    <script src="my_vega.js" type="text/javascript"></script>
+    <script src="my_vega_lite.js" type="text/javascript"></script>
+    <script src="my_vega_embed.js" type="text/javascript"></script>
+  </head>
+  <body>
+    {plot_divs}
+  </body>
+</html>
+```
+
+Note that this is a standard HTML file with only `{plot_divs}` as a placeholder
+for the generated plots. `<script>` tags in this file point to the local
```

Comment:
How about "for DVC to inject the plots."

Reply:
Also, the `<scripts>` part is pretty obvious, I think? Maybe mention it in the paragraph before the HTML code block.